CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 cmake_minimum_required(VERSION 3.8)
 
-project(Thrust CXX)
+project(Thrust NONE)
 
 set(THRUST_SOURCE ${CMAKE_SOURCE_DIR})
-include(cmake/common_variables.cmake)
+set(THRUST_FILECHECK_DATA_PATH "${THRUST_SOURCE}/internal/test")
```

Comment:
This is wrong, please put it back where it was. `common_variables.cmake` is also used by thrust/cmake/run_example.cmake. There's a reason why it was like it was.

Reply:
What's the reason?

Done.